testapi.pm

```diff
@@ -903,7 +903,7 @@ sub script_sudo {
 
 =head2 script_output
 
-  script_output($script [, $wait, type_command => 1])
+  script_output($script [, $wait, type_command => 1, proceed_on_failure => 0])
```

Comment:
this should be => 1 to be consistent with the type_command where we list the non-default usage